### PR TITLE
DOCSUP-18717: Update ClickHouse keeper

### PR DESCRIPTION
**Pull request by karnevil13**
Changelog category:

- Documentation for #24448 #31122

### CODE REVIEW

**Comment by alesapin** [comment]
@mergifyio update

File: docs/en/operations/clickhouse-keeper.md [hunk 1]
```diff
@@ -16,6 +16,11 @@ ZooKeeper is one of the first well-known open-source coordination systems. It's
 
 By default, ClickHouse Keeper provides the same guarantees as ZooKeeper (linearizable writes, non-linearizable reads). It has a compatible client-server protocol, so any standard ZooKeeper client can be used to interact with ClickHouse Keeper. Snapshots and logs have an incompatible format with ZooKeeper, but `clickhouse-keeper-converter` tool allows to convert ZooKeeper data to ClickHouse Keeper snapshot. Interserver protocol in ClickHouse Keeper is also incompatible with ZooKeeper so mixed ZooKeeper / ClickHouse Keeper cluster is impossible.
 
+ClickHouse Keeper supports Access Control List the same way as [ZooKeeper](https://zookeeper.apache.org/doc/r3.1.2/zookeeperProgrammers.html#sc_ZooKeeperAccessControl) does. ClickHouse Keeper supports the same set of permissions and has the identical built-in schemes: `world`, `auth`, `digest`, `host` and `ip`. Digest authentication scheme uses pair username:password. Password is encoded in Base64.
```
Comment (olgarev) [inline docs/en/operations/clickhouse-keeper.md:19]:
```suggestion
ClickHouse Keeper supports Access Control List (ACL) the same way as [ZooKeeper](https://zookeeper.apache.org/doc/r3.1.2/zookeeperProgrammers.html#sc_ZooKeeperAccessControl) does. ClickHouse Keeper supports the same set of permissions and has the identical built-in schemes: `world`, `auth`, `digest`, `host` and `ip`. Digest authentication scheme uses pair `username:password`. Password is encoded in Base64. 
```

File: docs/ru/operations/clickhouse-keeper.md [hunk 2]
```diff
@@ -16,12 +16,17 @@ ZooKeeper — один из первых широко известных сер
 
 По умолчанию ClickHouse Keeper предоставляет те же гарантии, что и ZooKeeper (линеаризуемость записей, последовательная согласованность чтений). У него есть совместимый клиент-серверный протокол, поэтому любой стандартный клиент ZooKeeper может использоваться для взаимодействия с ClickHouse Keeper. Снэпшоты и журналы имеют несовместимый с ZooKeeper формат, однако можно конвертировать данные Zookeeper в снэпшот ClickHouse Keeper с помощью `clickhouse-keeper-converter`. Межсерверный протокол ClickHouse Keeper также несовместим с ZooKeeper, поэтому создание смешанного кластера ZooKeeper / ClickHouse Keeper невозможно.
 
+Система управления доступом (ACL) ClickHouse Keeper реализована точно так же, как в [ZooKeeper](https://zookeeper.apache.org/doc/r3.1.2/zookeeperProgrammers.html#sc_ZooKeeperAccessControl). ClickHouse Keeper поддерживает тот же набор разрешений и имеет идентичные схемы: `world`, `auth`, `digest`, `host` и `ip`. Digest для аутентификации использует пару значений username:password. Пароль кодируется в Base64.
```
Comment (olgarev) [inline docs/ru/operations/clickhouse-keeper.md:19]:
```suggestion
Система управления доступом (ACL) ClickHouse Keeper реализована так же, как в [ZooKeeper](https://zookeeper.apache.org/doc/r3.1.2/zookeeperProgrammers.html#sc_ZooKeeperAccessControl). ClickHouse Keeper поддерживает тот же набор разрешений и идентичные схемы: `world`, `auth`, `digest`, `host` и `ip`. Digest для аутентификации использует пару значений `username:password`. Пароль кодируется в Base64.
```

File: docs/en/operations/clickhouse-keeper.md [hunk 3]
```diff
@@ -105,196 +109,6 @@ ClickHouse Keeper is bundled into the ClickHouse server package, just add config
 clickhouse-keeper --config /etc/your_path_to_config/config.xml --daemon
 ```
 
-## Four Letter Word Commands
```
Comment (alesapin) [inline docs/en/operations/clickhouse-keeper.md:108]: Why this part removed?

File: docs/en/operations/clickhouse-keeper.md [hunk 4]
```diff
@@ -49,7 +54,6 @@ Internal coordination settings are located in `<keeper_server>.<coordination_set
 -    `auto_forwarding` — Allow to forward write requests from followers to the leader (default: true).
 -    `shutdown_timeout` — Wait to finish internal connections and shutdown (ms) (default: 5000).
 -    `startup_timeout` — If the server doesn't connect to other quorum participants in the specified timeout it will terminate (ms) (default: 30000).
--    `four_letter_word_white_list` — White list of 4lw commands (default: "conf,cons,crst,envi,ruok,srst,srvr,stat,wchc,wchs,dirs,mntr,isro").
```
Comment (alesapin) [inline docs/en/operations/clickhouse-keeper.md:52]: Why it's removed?

**Review by alesapin — CHANGES_REQUESTED** [review]

File: docs/en/operations/clickhouse-keeper.md (hunk 3 above)
Comment (karnevil13) [reply docs/en/operations/clickhouse-keeper.md:108]: Sorry, no idea. I never touched anything apart from my task. And those lines weren't part of it.

File: docs/en/operations/clickhouse-keeper.md (hunk 4 above)
Comment (karnevil13) [reply docs/en/operations/clickhouse-keeper.md:52]: The same.

**Pull request merged**